cypress/integration/pages/articles/testsForCanonicalOnly.js

```diff
@@ -88,6 +88,28 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
             }
           });
         });
+
+        it('plays media when a user clicks play', () => {
+          cy.window().then(win => {
+            const media = getBlockData('video', win.SIMORGH_DATA.pageData);
+            if (media && media.type === 'video') {
+              cy.get(
+                'div[class^="StyledVideoContainer"] img[class^="StyledImg"]',
+              )
+                .click()
```

Comment:
@simonsinclair, one question, the steps on the [issue](https://github.com/bbc/simorgh/issues/4242) states
> When the user clicks on the play button (Clicked)

Why are we clicking on the placeholder image instead ?

Reply:
1. The `click` event that triggers playback can originate from anything below the Placeholder container.
2. The current element selection means we can check for it's removal with a chained `should('not.exist')` without re-selecting.